x-pack/packages/kbn-ai-assistant/src/buttons/ask_assistant_button.tsx

```diff
@@ -39,19 +39,20 @@ export type AskAssistantButtonProps = (
   onClick: () => void;
 };
 
+const AI_ASSISTANT_LABEL = i18n.translate('xpack.aiAssistant.aiAssistantLabel', {
+  defaultMessage: 'AI Assistant',
+});
```

Comment:
we're trying to avoid global `i18n.translate` evaluations because of [#180312](https://github.com/elastic/kibana/issues/180312). Do you mind moving it inside the function?

Reply:
TIL, done :)